.travis.yml

```diff
@@ -117,6 +117,17 @@ jobs:
         - travis_retry ./scripts/pod_lib_lint.rb FirebaseInstanceID.podspec --platforms=ios --use-modular-headers
         - travis_retry ./scripts/pod_lib_lint.rb FirebaseInstanceID.podspec --platforms=tvos --use-modular-headers
         - travis_retry ./scripts/pod_lib_lint.rb FirebaseInstanceID.podspec --platforms=macos --use-modular-headers
+ 
+    - stage: test
+      if: type = cron
```

Comment:
Delete this line so that travis runs on presubmits.

Also using travis is deprecated in favor of GitHub Actions.  See examples in .github/workflows